test/transforms/test_replicate.py

```diff
@@ -0,0 +1,262 @@
+from itertools import count
+from pprint import pprint
+
+import pytest
+from taskgraph.util.templates import merge
+
+from mozilla_taskgraph.transforms.replicate import transforms as replicate_transforms
+
+TC_ROOT_URL = "https://tc-tests.example.com"
+
+
+@pytest.fixture
+def get_target_defs():
```

Comment:
Is there a reason that this and `get_expected` are fixtures rather than regular functions? Seeing as the outer functions do nothing except return the inner ones, it seems unnecessary. I guess there might be value-add in having them listed in the test function arguments, but I can't find another reason.